totalopenstation/tests/test_nikon.py

```diff
@@ -17,8 +17,8 @@ def test_points(self):
     def test_basepoint(self):
         self.assertAlmostEqual(self.fp.points[0].geometry.x, 0.0)
         self.assertAlmostEqual(self.fp.points[0].geometry.y, 0.0)
-        self.assertAlmostEqual(self.fp2.points[7].geometry.x, 9936.99390187)
-        self.assertAlmostEqual(self.fp2.points[7].geometry.y, 10036.06473563)
+        self.assertAlmostEqual(self.fp2.points[7].geometry.x, 10000.08905808)
```

Comment:
This is a slight change, but do we have a way to check what it the correct value without adjusting tests? E.g. with known sample data that is processed by another "trusted" software.
No need to change here however!